test/e2e/framework/util.go

```diff
@@ -192,6 +191,7 @@ const (
 )
 
 var (
+	BusyBoxImage = imageutils.GetBusyBoxImage()
```

Comment:
I'm not sure we need to have both `framework.BusyBoxImage` and `common.busyBoxImage`. maybe consolidate in a follow-up?

Reply:
fixed it.

Reply:
sure, will do that later.